.evergreen/mongosh_dl.py

```diff
@@ -40,7 +42,7 @@ def _get_latest_version():
     url = "https://api.github.com/repos/mongodb-js/mongosh/releases"
     req = urllib.request.Request(url, headers=headers)
     try:
-        resp = urllib.request.urlopen(req, context=SSL_CONTEXT)
+        resp = urllib.request.urlopen(req, context=SSL_CONTEXT, timeout=10)
```

Comment:
```suggestion
        resp = urllib.request.urlopen(req, context=SSL_CONTEXT, timeout=30)
```
`mongodl.py` uses `timeout=30`. Should `mongosh_dl.py` also use `timeout=30`?